docs/integrations/keras.rst

```diff
@@ -1,112 +1,319 @@
+
+.. code-block:: python3 
+
+   import neptune
+   neptune.init(api_token='ADD-API-TOKEN-HERE',project_qualified_name='risingodegua/spam-classifier')
+
+The code above will initialize your project, and connect your local neptune client to your online account. So that any logs made locally can be synced online.
+
+.. Note::
+   Your API token is a secret, and should be hidden from outsiders. In production environments, it is advisable to add the token to your environment variable. Neptune will automatically pick up the variable from the environment as long as it is called NEPTUNE_API_TOKEN, so you can easily initialize Neptune by running:
+
+.. code-block:: python3
+
+   neptune.init(project_qualified_name='risingodegua/spam-classifier')
+
+
+
+**STEP 5**
+
+Create your first experiment under your project
 
-Installation
-^^^^^^^^^^^^
 .. code-block:: bash
 
-    pip install neptune-contrib
+   neptune.create_experiment(name='spam_exploration')
+
+.. image:: ../_static/images/integrations/keras-integration/out1.png
+
+
+Experiments are important for tracking and experimenting on different ideas, and aids reproduction. It is important to always create a new experiment for any new idea you want to try out.
+The |experiment link| provided will open the experiment dashboard where you can see details about your experiments in real time. This link can be shared with collaborators and even the general public if the project has been made public.
+Note: Neptune also tracks hardware metrics to tell you how your machine operates during experimentation. If you're running this experiment locally, you may have to install psutil library first.
+
+.. |experiment link| raw:: html
+
+    <a href="https://ui.neptune.ai/risingodegua/spam-classifier/e/SPAM-2" target="_blank">experiment link</a>
+
+.. code-block:: bash
+
+   pip install psutil
+
+To see your hardware metrics, you can click on the project |link| generated by Neptune above, and then select logs. You should be presented with a realtime dashboard as shown below:
+
+.. image:: ../_static/images/integrations/keras-integration/monitoring.png
+
+
+
+**Step 6**
+In a new cell of your notebook, import all the necessary packages needed for this project.
+
+.. code-block:: python3
+
+   ...
+   ...
+   from neptunecontrib.monitoring.keras import NeptuneMonitor
+
+Notice that we're importing the NeptuneMonitor from the neptunecontrib package? This is because we are going to be integrating neptune with keras. Neptune can also be integrated with other libraries easily. You can find some useful link on how to achieve this below.
+
+.. **TODO: SOME SEO LINKS HERE**
+
+
+Now since this is a text classification dataset, there are numerous preprocessing steps you will follow in order to get your data into numeric form which you can pass to your model. We assume you have performed these steps and have the saved some important parameters like vocabulary size, input size and so on. 
```

Comment:
This whole section focuses on the text classificaiton dataset -> but it should just show keras integration in mind. 
This is making it fuzzy -> could you please change that?